src/tools/run-make-support/src/command.rs

```diff
@@ -151,6 +151,53 @@ impl Command {
         self
     }
 
+    /// Set an auxiliary stream passed to the process, besides the stdio streams.
+    ///
+    /// # Safety
+    ///
+    /// Use with caution! Ideally, only set one aux fd; if there are multiple, their old `fd` may
```

Comment:
nit: I would write this in a way that more clearly states the requirement. though given that the safety requirement is basically impossible to actually uphold (see next comment) I would probably just make this function safe because the unsafe is just noise

Reply:
Good point, I just dropped the unsafe.